src/app/service/shoppingcart.service.ts

```diff
@@ -53,4 +46,29 @@ export class ShoppingCartService {
   loadCart() {
     this.cart = JSON.parse(localStorage.getItem('cart'));
   }
+
+  clearCart() {
+    this.cart = [];
+    this.saveCart();
+  }
+
+  removeProduct(p_id: string, p_name: string, price: number, i_id: string, i_name: string) {
```

Comment:
sorry, but the argument names are confusing

Reply:
Will check it out